cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortServiceExceptionMapper.java

```diff
@@ -88,6 +89,11 @@ else if (ex instanceof MismatchedInputException) {
 				serviceErrorCode = Status.BAD_REQUEST.getStatusCode();
 				serviceErrorListCode = serviceErrorCode;
 			}
+			else if (ex instanceof JsonParseException) {
```

Comment:
I added a case for MismatchedInputException above which is a Jackson data bind exception I encountered in my testing. You have a block for JsonParseException. Both are parented by JsonProcessingException. I wonder if we should just handle both your case and mine in a JsonProcessingException block.

Reply:
I'm not totally opposed to it, but just added the section for one I strictly encountered like you did. Not sure what other kind of exceptions that could get generated that would fall under the parent. I think if testing uncovers more, then we could change it.